commons-packet/commons-packet-manager/src/main/java/io/mosip/commons/packet/impl/OfflinePacketCryptoServiceImpl.java

```diff
@@ -103,7 +103,8 @@
         cryptomanagerRequestDto.setSalt(CryptoUtil.encodeBase64String(nonce));
         // setLocal Date Time
         if (id.length() > 14) {
-            String packetCreatedDateTime = id.substring(id.length() - 14);
+        	String packetRid = id.split("-")[0];
+            String packetCreatedDateTime = packetRid.split("-")[0].substring(packetRid.length() - 14);
```

Comment:
u have already split with ("-"). Why are u doing again in line no 107. Relook into this code.

Reply:
fixed